engine/time/src/main/java/io/deephaven/time/DateTimeUtils.java

```diff
@@ -334,6 +334,26 @@ public static Instant now() {
         return currentClock().instantNanos();
     }
 
+    @ScriptApi
```

Comment:
no docs

Reply:
Arguably we could take this out of the API in Python, instead of adding a new Java method. Presumably if we thought it was really necessary, we would already have a Java method like this. `TimeUnit` is a much better way to express this, rather than a String.